tests/rules/test_WildcardResourceRule.py

```diff
@@ -661,16 +683,38 @@ def test_multiple_resources_with_wildcard_resources_are_detected(user_and_policy
                 rule="WildcardResourceRule",
                 rule_mode="BLOCKING",
                 actions={
-                    "dynamodb:CreateTable",
-                    "dynamodb:BatchGet*",
-                    "dynamodb:Scan",
                     "dynamodb:Update*",
+                    "dynamodb:DescribeStream",
+                    "dynamodb:BatchGet*",
+                    "dynamodb:CreateTable",
                     "dynamodb:Query",
                     "dynamodb:Delete*",
+                    "dynamodb:BatchWrite*",
+                    "dynamodb:DescribeTable",
+                    "dynamodb:Scan",
                     "dynamodb:PutItem",
+                    "dynamodb:Get*",
+                },
+                resource_ids={"RolePolicy"},
+                resource_types={"AWS::IAM::Policy"},
+            ),
+            Failure(
```

Comment:
Two new failures are added. Review if they are correct

Reply:
They are, they come from https://github.com/Skyscanner/pycfmodel/pull/133 which is not included in the latest release yet